File: Python/Kode Program/Pasta.py
from Items import Items

class Pasta(Items):
    def __init__(self, Id="", Nama="", Kategori="", HargaProduk=0, Saus="", JenisPasta=""):
        super().__init__(Id, Nama, Kategori, HargaProduk)
        self.__Saus = Saus
        self.__JenisPasta = JenisPasta

    def set_saus(self, saus1):
        self.__Saus = saus1

    def get_saus(self):
        return self.__Saus

    def set_pasta(self, pasta1):
        self.__JenisPasta = pasta1

    def get_pasta(self):
        return self.__JenisPasta

File: CPP/Kode Program/Pasta.cpp
#ifndef PASTA_CPP
#define PASTA_CPP

#include <iostream>
#include <string>
#include "Items.cpp"

using namespace std;

class Pasta : public Items
{
    private:
        string Saus;
        string JenisPasta;

    public:
        // Constructor Default
        Pasta() 
        {

        }

        // Setter dan Getter Saus
        void setSaus(string saus1)
        {
            Saus = saus1;
        }

        string getSaus()
        {
            return Saus;
        }

        // Setter dan Getter Pasta
        void setPasta(string pasta1)
        {
            JenisPasta = pasta1;
        }

        string getPasta()
        {
            return JenisPasta;
        }

        ~Pasta()
        {

        }
};

#endif  // SPAGETTI_CPP